stories/splash.stories.js

```diff
@@ -37,7 +37,7 @@ storiesOf( 'Splash', module )
 						">
 							<DashboardSplashModule
 								icon="search-console"
-								title={ __( 'Search Console', 'google-site-kit' ) }
+								title={ _x( 'Search Console', 'Service name', 'google-site-kit' ) }
```

Comment:
The `_x` function needs to be imported at the top of the file (in addition to `__`)

Reply:
Note: This applies to all the files where this change was made. I just didn't wanna mention it everywhere :-)